tests/main/pwd/task.yaml

```diff
@@ -0,0 +1,39 @@
+summary: current working directory is preserved (if possible)
+details: |
+    snap-confine will preserve the current working directory as long as that
+    directory is the same inside the execution environment. If the directory
+    exists but is not the same (e.g. /tmp) it will be re-interpreted according
+    to the view inside the mount namespace. If the directory does not exist the
+    special fallback /var/lib/snapd/void is used.
+prepare: |
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local test-snapd-sh
+    # NOTE: On core18 some directories are still missing. To decouple that fix
+    # from this test create the essential directory if it is missing.
+    test -d /var/lib/snapd/void || mkdir -m 111 /var/lib/snapd/void
```

Comment:
shouldn't we check  that we are on core18 and not do this uncoditionally?

Reply:
I will revert this part. It will be fixed in the proper place soon.

Reply:
This is gone now.